libpod/container.go

```diff
@@ -106,6 +107,9 @@ type Container struct {
 	valid   bool
 	lock    storage.Locker
 	runtime *Runtime
+
+	rootlessSlirpSyncR *os.File
```

Comment:
Do we need to restore these when the container is retrieved from the database? Or do we just need them to pass into conmon, and we don't ever need them again?

Reply:
we need only to pass them into conmon and leak the fd.  When the conmon ends is closed slirp4netns will detect that and also exits, so we won't need to cleanup the process separately.  This is the relevant change in slirp4netns to do that: https://github.com/rootless-containers/slirp4netns/commit/76d4ceff9ecfd0f0dfc25afe7389a00c65fe9e12